sample/Kmd.Logic.FileSecurity.Client.ConfigurationSample/Program.cs

```diff
@@ -104,7 +104,15 @@ private static async Task Run(AppConfiguration configuration)
                 return;
             }
 
-            Console.WriteLine("Detailed Sign Configuration data: {@SignConfiguration}", signConfigurationGetResult);
+            Console.WriteLine(
+                "Sign Configuration Details. \nSignConfiguration ID: {0} \nSignConfiguration Name: {1}\nCertificate ID : {2}\nSubscription ID : {3}\nCreatedDate : {4}\nDocumentType : {5}\nPrivileges : {6}",
```

Comment:
Maybe you can write in multiple lines for readability purpose.

Reply:
made changes as suggested